engine/core/src/test/java/org/datacleaner/util/ReflectionUtilsTest.java

```diff
@@ -154,6 +155,20 @@ public void testGetFields() throws Exception {
         assertEquals("a", fields[1].getName());
     }
 
+    private Field[] getNonSyntheticFields(Class clazz) {
```

Comment:
This change is now "fixing" something in the test. But it seems to me that maybe it is ReflectionUtils itself that needs this fix?